### ISSUE TITLE
Require marbl_status_log in drtsafe()

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#21]
FIXME comment in `marbl_co2calc_mod::drtsafe`:

```
make marbl_status_log required - this is currently needed
since abio_dic_dic14_mod is calling this routine but has
not itself been MARBLized yet
```

**Comment by mnlevy1981** [comment]
`marbl_status_log` is still optional in `marbl_co2calc_mod::drtsafe`, but the routine is private so it's not being used by `abio_dic14_mod`; I think the optional attribute of the variable can be safely removed.

**Comment by mnlevy1981** [comment]
closed via #178
